nixos/modules/system/boot/systemd.nix

```diff
@@ -136,6 +139,16 @@ let
       "systemd-ask-password-wall.path"
       "systemd-ask-password-wall.service"
 
+      # Varlink APIs
+      "systemd-bootctl@.service"
```

Comment:
No docs; https://raw.githubusercontent.com/systemd/systemd/main/units/systemd-bootctl%40.service is the service itself. Release notes mention:

> bootctl now provides a basic Varlink interface and can be run as a daemon via a template unit.